timelock-server-benchmark-client/src/main/java/com/palantir/atlasdb/timelock/benchmarks/AbstractBenchmark.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.timelock.benchmarks;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Throwables;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+public abstract class AbstractBenchmark {
+
+    private static final Logger log = LoggerFactory.getLogger(AbstractBenchmark.class);
+
+    private final int numClients;
+    private final int requestsPerClient;
+
+    private final ExecutorService executor;
+    private final List<Long> times = Lists.newArrayList();
```

Comment:
We might want to wrap this with Collections.synchronizedList (or use a j.u.c collection), see below